libavfilter/dnn/dnn_backend_native.c

```diff
@@ -45,9 +45,29 @@ static const AVClass dnn_native_class = {
     .category   = AV_CLASS_CATEGORY_FILTER,
 };
 
-static DNNReturnType execute_model_native(const DNNModel *model, const char *input_name, AVFrame *in_frame,
-                                          const char **output_names, uint32_t nb_output, AVFrame *out_frame,
-                                          int do_ioproc);
+static DNNReturnType execute_model_native(Queue *inference_queue);
+
+static DNNReturnType extract_inference_from_task(TaskItem *task, Queue *inference_queue)
```

Comment:
fyi, we may change the inference in a new patch after this patch set is finished, it is really confusing.

the struct name can be LastLevelTaskItem, and the function name may extract_lltask_from_task, we use lltask for variable.

The last level task means the task can be passed to the inference engine, so, 'task' for the interface between filter and ffmpeg-dnn, while lltask is the interface between dnn and inference engine.

Reply:
Sure, got the point. Shall I include this in this patchset as the last patch or include it in the next patch set?

Reply:
add as the last patch would be better

Reply:
Okay, I'll add the patch today.

Reply:
What about the `inference_queue`? Shall I name it `lltask_queue`?

Reply:
yes. right.

Reply:
Okay, thanks.

Reply:
Added